packages/core/src/util/renderer.ts

```diff
@@ -946,7 +957,6 @@ export const mapStateToCombinatorRendererProps = (
 
   const ajv = state.jsonforms.core.ajv;
   const schema = resolvedSchema || rootSchema;
```

Comment:
This `schema` is the one which we hand over to the renderer. In case of resolving, this will always be a new instance and therefore break memorization. We should hand over an unchanged schema to the renderer and then deal with the consequences.

Reply:
I reverted the change.